src/main/java/com/gregtechceu/gtceu/data/recipe/misc/VanillaStandardRecipes.java

```diff
@@ -701,6 +735,24 @@ private static void redstoneRecipes(Consumer<FinishedRecipe> provider) {
                 .inputFluids(Concrete.getFluid(L))
                 .outputItems(new ItemStack(Items.COMPARATOR))
                 .duration(100).EUt(1).save(provider);
+
+        if (!ConfigHolder.INSTANCE.recipes.hardRedstoneRecipes) {
```

Comment:
same question, hard configs still have these recipes?

Reply:
Not sure if I should add cutting recipes for these since they use springs in hard configs